### ISSUE TITLE
Statistical and Machine Learning concepts 

### ISSUE THREAD

**Issue by Starsa** [Starsa/Mentorship#28]

**Comment by Starsa** [comment]
**_p-value:_**

> the probability of getting results at least as extreme as what we observed, given the null hypotheses is true.

**Comment by Starsa** [comment]
**_R2_**

> How much variance in the target vector is explained by the features. Determines goodness of fit.

**Comment by Starsa** [comment]
**_Adjusted R2_**

> Modified version of R2, it imposes a penalty for too many features

**Comment by Starsa** [comment]
**_AUC_**

> AUC (Area under the ROC curve) is an aggregate measure of performance across all possible classification thresholds.

**Comment by Starsa** [comment]
### _Precision_

> Measure of accuracy (True Positive/ TruePositives + False Positives)

### _Recall_

> True positive rate (True Positives/ TruePositives + False Negatives)
>> _if we wanted to find all positives, absolutely, maximize the recall_

**Comment by Starsa** [comment]
### _Bias_

> Error- Expected error created by using a model to approximate a real-world function

### _Variance_

> Noise- The error from sensitivity to small fluctuations in the training set

### _Bias Variance Trade-off_

> A simple model has high bias and low variance and a complex model has high variance and low bias. The tradeoff is finding a good balance without overfitting or underfitting the data.

**Comment by CLuiz** [comment]
These are all great. Hopefully you won't get a grumpy statistician asking about p values

**Comment by CLuiz** [comment]
> **_AUC_**
> 
> > AUC (Area under the ROC curve) is an aggregate measure of performance across all possible classification thresholds.

Can you work out why I don't like AUC for model evaluation from this definition?

**Comment by Starsa** [comment]
> ### _Precision_
> > Measure of accuracy (True Positive/ TruePositives + False Positives)
> 
> ### _Recall_
> > True positive rate (True Positives/ TruePositives + False Negatives)
> > > _if we wanted to find all positives, absolutely, maximize the recall_

According to Cassie Kozyrkov:
Precision: "Don't waste my time. Missed opportunities are okay"
Recall: "Collect 'em all. Duds are okay"

**Comment by Starsa** [comment]
### Accuracy
> "All mistakes are equally bad"

### F-measure
> "I can't choose between precision and recall"
